nuget/Directory.Build.props

```diff
@@ -12,7 +12,8 @@
 
     <!-- Distro rid is passed as runtimeos-arch-->
     <_parseDistroRid>$(__DistroRid)</_parseDistroRid>
-    <_parseDistroRid Condition="'$(_parseDistroRid)' == '' and '$(__BuildOS)' == 'OSX'">osx.10.12-x64</_parseDistroRid>
+    <_parseDistroRid Condition="'$(_parseDistroRid)' == '' and '$(__BuildOS)' == 'OSX' and '$(Platform)' == 'arm64'">osx.11.0-arm64</_parseDistroRid>
+    <_parseDistroRid Condition="'$(_parseDistroRid)' == '' and '$(__BuildOS)' == 'OSX' and '$(Platform)' != 'arm64'">osx.10.12-x64</_parseDistroRid>
```

Comment:
@directhex given we use `CMAKE_OSX_DEPLOYMENT_TARGET=10.13` we should probably bump the RID to `osx.10.13-x64`? or actually why do we use a version-specific RID at all rather than `osx-x64` like we do elsewhere?

Reply:
That is an EXCELLENT QUESTION for past Jo, because present day Jo is experiencing what a pain in the ass it is to consume it this way, given how llvm-init.proj is written